app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'prawn'
+require 'prawn/table'
+require 'lighthouse/veterans_health/client'
+
+module FastTrack
+  class DisabilityCompensationJob
+    include Sidekiq::Worker
+
+    extend SentryLogging
+    # TODO: This is apparently at most about 4.5 hours.
```

Comment:
Why is this a TODO?

Reply:
Should have been `Note`; fixed in https://github.com/department-of-veterans-affairs/vets-api/commit/ece5c067066954b9b0090211b8cf6af2ceb06ca4